src/services/ui/src/utils/user.ts

```diff
@@ -17,10 +17,22 @@ export const isAuthorizedState = async (id: string) => {
     const user = await getUser();
     if (!user.user) throw Error("No cognito attributes found.");
     return getUserStateCodes(user.user).includes(
-      id.substring(0, 2) as StateCode
+      id.substring(0, 2) as StateCode,
     );
   } catch (e) {
     console.error(e);
     return false;
   }
 };
+
+export const authorityById = (id: string) => {
```

Comment:
right now `Authority` can be Med SPA, CHIP SPA, 1915b, or 1915c. this is only concerned with a binary value of spa/waiver which sounds like we could solve it via boolean logic instead. was there a future case you considered where this got more granular with authority, or where new categories with alternative ids are added?

Reply:
Not particularly, however the name could be improved. I'm using it to plug into the tab query parameter. So maybe, tab it could be dashboardTabById? I'm open to using a boolean though as well. What would your ideal call be for this if you wrote it. Just want to gauge where your head is at with it. I may even steal your idea lol

Reply:
i think ID measurement is fine, i just think we can skip some of the if/else logic and return `spaBegin.test(id)` and name it like `toSpaTab` or something. that way we're leaving the bones of your id measurement logic in place in case we _do_ have a future need to expand (i.e. a 3rd tab)